docs/dev-guide/contributing/bugs.rst

```diff
@@ -46,6 +46,17 @@ necessary, the bug may be commented on requesting more information or
 clarification from the reporter. When a bug has enough information, its
 *Priority* rating set and is marked as triaged using the *Triaged* boolean.
 
+Blocking Bugs
+-------------
+
+All Redmine Issues have a "Blocks Release" field. This field refers to upcoming
+Platform Release versions that *cannot* be released until this bug is fixed.
+
+For example, if a blocking issues is discovered in Pulp 2.4.6, and 2.4.7 is the next Platform
+Release version, then the issue's "Blocks Release" field should be set to ``2.4.z``. If the bug
+also affects the next Platform Release of Pulp, such as 2.5.2, the issue's "Block Release" field
+should also include ``2.5.z``.
+
```

Comment:
Yeah, that makes sense. I've tried a couple of times to put something down, but I feel like the python situation is a unique one and not a pattern we'll necessarily follow in the future.